src/test/lib/ApiTest.cpp

```diff
@@ -1735,6 +1735,65 @@ class SecConfigTestContext {
     }
 };
 
+void QuicTestSetParam()
```

Comment:
So this function doesn't get huge in subsequent PRs, consider renaming this to `QuicTestGlobalSetParam` and use it to test the global parameter space.
Or name it `QuicTestParamLevelSetParam` if you just want to do the same tests on all different param levels.

Reply:
I adopted idea `QuicTestGlobalSetParam`